### PR TITLE
add evictionStartTime for appliedmanifestwork

### PR DESCRIPTION
**Pull request by skeeey**

### CODE REVIEW

File: work/v1/types.go [hunk 1]
```diff
@@ -545,6 +545,15 @@ type AppliedManifestWorkStatus struct {
 	// However, the resource will not be undeleted, so it can be removed from this list and eventual consistency is preserved.
 	// +optional
 	AppliedResources []AppliedManifestResourceMeta `json:"appliedResources,omitempty"`
+
+	// EvictionStartTime represents the current appliedmanifestwork will be evicted after a grace period.
+	// An appliedmanifestwork will be evicated from the managed cluster in the following scenarios:
```
Comment (qiujian16) [inline work/v1/types.go]: typo evicted

File: work/v1/types.go [hunk 2]
```diff
@@ -545,6 +545,15 @@ type AppliedManifestWorkStatus struct {
...
+	// - the appliedmanifestwork hub hash is unchanged, and its manifestwork is not found from the hub (e.g.
+	//   the klusterlet is reconnected to the hub, or the klusterlet is rejoined) to the hub), or
+	// - the appliedmanifestwork hub hash is changed and its agent ID is unchanged (e.g. the hub of current klusterlet
```
Comment (qiujian16) [inline work/v1/types.go]: I think the main idea is
Any appliedmanifestworks that has no matched manifestworks on the hub will be evicted. The scenario includes:
1. manifestwork is missing on the hub
2. the hubhash of the appliedmanifestwork does not match the current hubhash.

File: work/v1/types.go (hunk 1 above)
Comment (skeeey) [reply work/v1/types.go]: corrected

File: work/v1/types.go (hunk 2 above)
Comment (skeeey) [reply work/v1/types.go]: make it clear

File: work/v1/types.go [hunk 3]
```diff
@@ -545,6 +545,13 @@ type AppliedManifestWorkStatus struct {
 	// However, the resource will not be undeleted, so it can be removed from this list and eventual consistency is preserved.
 	// +optional
 	AppliedResources []AppliedManifestResourceMeta `json:"appliedResources,omitempty"`
+
+	// EvictionStartTime represents the current appliedmanifestwork will be evicted after a grace period.
+	// An appliedmanifestwork will be evicted from the managed cluster in the following two scenarios:
+	//   - the manifestwork of the current appliedmanifestwork is missing on the hub, or
+	//   - the appliedmanifestwork hub hash does not match the current appliedmanifestwork hub hash.
```
Comment (qiujian16) [inline work/v1/types.go]: the appliedmanifestwork hub hash does not match the current hub hash of the work agent.

**Review by qiujian16 — COMMENTED** [review]
/approve

just a minor comments on the description

File: work/v1/types.go (hunk 3 above)
Comment (skeeey) [reply work/v1/types.go]: updated

**Pull request merged**